validator/client/validator_aggregate.go

```diff
@@ -112,7 +115,12 @@ func (v *validator) signSlot(ctx context.Context, pubKey [48]byte, slot uint64)
 	if err != nil {
 		return nil, err
 	}
-	sig, err := v.keyManager.Sign(pubKey, root)
+	var sig *bls.Signature
+	if protectingKeymanager, supported := v.keyManager.(keymanager.ProtectingKeyManager); supported {
```

Comment:
Can we consolidate this into a shared helper func? it's repeated in a few places

Reply:
Updated; thanks.